src/finalize.F90

```diff
@@ -1,17 +1,14 @@
 module finalize
 
+  use hdf5, only: h5tclose_f, h5close_f
+
```

Comment:
Is this newline here for a reason? Any reason to split up the headers from `hdf5` and `hdf5_interface`?